mobile/packages/swap/adapters/api/minswap/transformers.ts

```diff
@@ -0,0 +1,364 @@
+          return freeze(
+            {
+              defaultProtocol: Swap.Protocol.Minswap_v2,
+              wantedPrice: parseFloat(data.price.toString()),
+              options: data.options.map((option) =>
+                freeze(
+                  {
+                    protocol: mapDexToProtocol(option.protocol),
+                    initialPrice: option.price,
+                    batcherFee: option.fee,
+                  },
+                  true,
+                ),
+              ),
+            },
+            true,
+          )
+        },
+      },
+
+      estimate: {
+        request: ({
+          amountIn,
+          blockedProtocols,
+          slippage,
+          tokenIn,
+          tokenOut,
+        }: Swap.EstimateRequest): EstimateRequest => {
+          const request: EstimateRequest = {
+            token_in: toTokenId(tokenIn),
+            token_out: toTokenId(tokenOut),
+            amount: amountIn?.toString() || '0',
+            slippage: slippage || 1,
+            exclude_protocols: blockedProtocols?.map((p) =>
+              mapProtocolToDex(p),
+            ),
+            amount_in_decimal: true, // Tell API that amounts are in decimal format
+            ...(partner !== undefined && {partner}),
+          }
+          return request
+        },
+        response: (data: EstimateResponse): Swap.EstimateResponse => {
+          const totalInput = parseFloat(data.amount_in)
+          const totalOutput = parseFloat(data.amount_out)
+          const totalOutputWithoutSlippage = parseFloat(data.min_amount_out)
+
+          return freeze(
+            {
+              splits: transformPathsToSplits(data.paths),
+              batcherFee: parseFloat(data.deposits || '0'),
+              deposits: parseFloat(data.deposits || '0'),
+              aggregatorFee: parseFloat(data.aggregator_fee || '0'),
+              frontendFee: 0,
+              netPrice: totalOutput / totalInput,
+              priceImpact: data.avg_price_impact,
+              totalFee: parseFloat(data.total_dex_fee || '0'),
+              totalOutput: totalOutput,
+              totalOutputWithoutSlippage: totalOutputWithoutSlippage,
+              totalInput: totalInput,
+            },
+            true,
+          )
+        },
+      },
+
+      create: {
+        request: ({
+          amountIn,
+          blockedProtocols,
+          slippage,
+          tokenIn,
+          tokenOut,
+        }: Swap.CreateRequest): CreateRequest => {
+          const request = {
+            sender: address,
+            min_amount_out: '0', // Will be calculated by the API
+            estimate: {
+              amount: amountIn.toString(),
+              token_in: toTokenId(tokenIn),
+              token_out: toTokenId(tokenOut),
+              slippage: slippage || 1,
+              exclude_protocols: blockedProtocols?.map((p) =>
+                mapProtocolToDex(p),
+              ),
+              ...(partner !== undefined && {partner}),
```

Comment:
if it can be null better use weak `!= null` otherwise null will be true
```suggestion
              ...(partner !== undefined && {partner}),
```

Reply:
It can't, it's coming from our config, it's either there or not there, we're never setting it to null